lib/iris/tests/__init__.py

```diff
@@ -183,7 +183,10 @@ def get_data_path(relative_path):
     """
     if not isinstance(relative_path, six.string_types):
         relative_path = os.path.join(*relative_path)
-    data_path = os.path.join(iris.config.TEST_DATA_DIR, relative_path)
+    test_data_dir = iris.config.TEST_DATA_DIR
+    if test_data_dir is None:
```

Comment:
I don't see the need for this..
From how ```iris.config.TEST_DATA_DIR``` is initialised in iris.config, I think this **can't** be empty ?